packages/@angular/cli/plugins/karma.ts

```diff
@@ -8,6 +8,14 @@ import { Pattern } from './glob-copy-webpack-plugin';
 import { WebpackTestConfig, WebpackTestOptions } from '../models/webpack-test-config';
 import { KarmaWebpackThrowError } from './karma-webpack-throw-error';
 
+/**
+ * Enumerate needed (but not require/imported) dependencies from this file
+ *  to let the dependency validator know they are used.
+ *
+ * require('karma-source-map-support')
+ */
```

Comment:
Yes, otherwise the dependency validator thinks this dependency is not used. It's needed but it's required via `require.resolve()` instead of `require()`, so the automatic detection does not work. The same workaround is used in `webpack-configs/common.ts` for loaders.